src/components/frontend-engine/types.ts

```diff
@@ -106,6 +107,7 @@ export type TFrontendEngineFieldSchema<V = undefined> =
 export type TFrontendEngineValues<T = any> = Record<keyof T, T[keyof T]>;
 export type TRevalidationMode = Exclude<keyof ValidationMode, "onTouched" | "all">;
 export type TValidationMode = keyof ValidationMode;
+export type TRestoreMode = "default-value" | "user-input";
```

Comment:
apologies, I missed this out, can we have another mode to not restore the values, this is to support our current behaviour of no restoring the values at all

Reply:
added a `"none"` mode which clears the values similarly to the reset button